Packet++/src/CotpLayer.cpp

```diff
@@ -0,0 +1,72 @@
+#include "../header/CotpLayer.h"
+#include "EndianPortable.h"
+#include <PayloadLayer.h>
+#include <cstring>
+#include <iostream>
+#include <sstream>
+
+namespace pcpp
+{
+
+	pcpp::CotpLayer::CotpLayer(uint8_t length, uint8_t pdu_type, uint8_t tpdu_number)
+	{
+		const size_t headerLen = sizeof(cotphdr);
+		m_DataLen = headerLen;
+		m_Data = new uint8_t[headerLen];
+		memset(m_Data, 0, headerLen);
+		cotphdr *cotpHdr = (cotphdr *)m_Data;
+		cotpHdr->length = length;
+		cotpHdr->pdu_type = pdu_type;
+		cotpHdr->tpdu_number = tpdu_number;
+		m_Protocol = COTP;
+	}
+
+	void CotpLayer::computeCalculateFields() {}
+
+	std::string CotpLayer::toString() const
+	{
+		std::ostringstream lengthStream;
+		lengthStream << std::to_string(getLength());
+		std::ostringstream pduTypeStream;
+		pduTypeStream << std::to_string(getPdu_type());
+		std::ostringstream tpduNumberStream;
+		tpduNumberStream << std::to_string(getTpdu_number());
+
+		return "Cotp Layer length: " + lengthStream.str() + ", pdu_type: " + pduTypeStream.str() +
+			   ", tpdu_number: " + tpduNumberStream.str();
+	}
```

Comment:
if we only support length of 2 and pdu type of `0x0f` then maybe we don't need all this information?
Can probably be just `COTP Layer`?